tools/trace-plotter/models/models.go

```diff
@@ -0,0 +1,66 @@
+package models
```

Comment:
why is it called models?

Reply:
It is for the elasticsearch response models, trace structure models and so